docs/manual/scala/gettingstarted/IntroGetStarted.md

```diff
@@ -20,6 +20,7 @@ To create your project, follow these steps:
    ```
    sbt -Dsbt.version=0.13.15 new https://github.com/lagom/lagom-scala.g8
    ```
+   Note: if you are on Windows and error shows: ```invalid system property 'sbt.version'```, then use the command with double quotes on -Dsbt.version so that it becomes as follows: ```sbt "-Dsbt.version=0.13.15" new https://github.com/lagom/lagom-scala.g8```
```

Comment:
I think it would be better to just insert the double quotes in the example command line, rather than complicating the instructions with multiple options. Bash definitely will not have any problem with them, so if it fixes a problem with the Windows command shell, then let's just put them there for everyone.